### ISSUE TITLE
Added lobster analysis

### ISSUE THREAD

**Issue by TimLanglois** [UWA-SCIE2204-Marine-Systems/No-take-marine-reserves#1]
hi @Matt-Navarro 
I have added simple lobster data and ANOVA.
What else does it need - or is this too complex?

**Comment by Matt-Navarro** [comment]
I think it is good! Do you have any other dependent variables to flesh out the story a bit?


________________________________

**Comment by TimLanglois** [comment]
Well – there is a VERY interesting relationship with the covar complexity – habitat complexity 1-5 rank .

But I though that might be too much for them?

Is a two factor interaction fnot enough? They should do pairwise comparison of Status for each location and then add the results to the plot!

**Comment by Matt-Navarro** [comment]
Yeah I was thinking along the lines of an alternate response variable (not sure what that might be). But,  that way we could just repeat the same code (keep it simple) but with a different response.

I think the habitat stuff might be getting a bit too complicated.

I'll try and flesh it out as is and let you know how I go.

Cheers,
Matt


________________________________
